e2e/cypress/tests/integration/messaging/edit_message_spec.js

```diff
@@ -145,4 +145,42 @@ describe('Edit Message', () => {
             cy.get(postText).should('have.text', `${secondMessage} Another new message Edited`);
         });
     });
+
+    it('should discard any changes made after cancelling the edit and opening the edit textbox again should display the original message', () => {
```

Comment:
@furqanmlk cloud you please help with the test id here

Reply:
@furqanmlk is it no required

Reply:
Hey @AshishDhama, Sorry for the late response, In order to get the `test ID`, you need to create a PR in test management repo `https://github.com/mattermost/mattermost-test-management`.
For example https://github.com/mattermost/mattermost-test-management/pull/38

Reply:
@gitstart can you add the test id here